modules/mod_ginger_base/translations/nl.po

```diff
@@ -32,18 +32,26 @@ msgstr "1 reactie"
 msgid "1 person interested"
 msgstr "1 geïnteresseerde"
 
+#: user/modules/mod_ginger_base/templates/pager/pager.tpl:3
+msgid "About {n} results"
+msgstr "Ongeveer {n} resultaten"
+
 #: user/modules/mod_ginger_base/templates/about/about.tpl:1
 msgid "About:"
 msgstr "Over: "
 
-#: user/modules/mod_ginger_base/templates/copyrights/copyrights.tpl:52
+#: user/modules/mod_ginger_base/templates/_admin_edit_content_address.tpl:5
+msgid "Address"
+msgstr "Adres"
+
 #: user/modules/mod_ginger_base/templates/copyrights/edit-copyrights.tpl:81
+#: user/modules/mod_ginger_base/templates/copyrights/copyrights.tpl:51
 msgid "All rights reserved"
 msgstr "Alle rechten voorbehouden"
 
 #: user/modules/mod_ginger_base/templates/date/edit-month.tpl:16
-msgid "April"
-msgstr ""
+msgid "Apr"
```

Comment:
Zijn de maanden nu bewust afgekort?

Reply:
Die `msgid` komt ergens uit een template, dus dat zal idd wel bewust zijn gedaan door iemand.  Alleen is er een tijdje kennelijk geen nieuwe .po file gemaakt.